src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -120,6 +126,8 @@ internal static ErrorCode GetErrorCode(this LanguageVersion version)
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7;
                 case LanguageVersion.CSharp7_1:
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7_1;
+                case LanguageVersion.CSharp7_2:
+                    return ErrorCode.ERR_FeatureNotAvailableInVersion7_2;
```

Comment:
I'd raised that question before. Someone (maybe Vlad or Neal) had previously suggested this helps telemetry.
Filed follow-up issue https://github.com/dotnet/roslyn/issues/19871